packages/react/src/analytics/integrations/shared/GoogleConsentMode/GoogleConsentMode.ts

```diff
@@ -5,42 +5,18 @@ import { type GoogleConsentModeMappings } from './types.js';
  * GoogleConsentMode handles with Google Consent Mode v2.
  */
 export class GoogleConsentMode {
-  private dataLayers!: Array<string>; // Stores different data layer names
-  private enabled!: boolean; // enable or disable CMP integration
+  private dataLayers!: string; // Stores different data layer names
```

Comment:
Since this is a single datalayer now, you can rename this to singular:

```suggestion
  private dataLayer!: string; // Stores different data layer names
```